modules/impersonation-system-admin-user.adoc

```diff
@@ -16,3 +16,16 @@ cluster administrator permissions.
 ----
 $ oc create clusterrolebinding <any_valid_name> --clusterrole=sudoer --user=<username>
 ----
+
+When a `system:admin` user is granted cluster administration permissions via a group, you must include the
+`*--as=<user> --as-group=<group1> --as-group=<group2>*` parameters in the command to impersonate the associated group(s).
+
+.Procedure
+
+* To grant a user permission to impersonate a `system:admin` by impersonating the associated cluster administration group(s),
+run the following command:
++
+[source,terminal]
+----
+$ oc create clusterrolebinding <any_valid_name> --clusterrole=sudoer --as=<user> \ --as-group=<group1> --as-group=<group2>
```

Comment:
Your command has a "\\" in the middle of it. The only reason that "\\" is used is to end a command, but continue it on the next line. So if you want that, then it should be laid out like this:

```
[source,terminal]
----
$ oc create clusterrolebinding <any_valid_name> --clusterrole=sudoer --as=<user> \ 
  --as-group=<group1> --as-group=<group2>
----
```

So you can either split it up like that, or just keep it all on one line and remove the "\\"